doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,139 @@
+
+
+### Astarte payload standard fields
+
+| Key | Type             | Mandatory | Description                               |
+|-----|------------------|-----------|-------------------------------------------|
+| v   | any Astarte type | Yes       | The property or the datastream sent value |
+| t   | UTC datetime     | No        | Datastream explicit value                 |
+
+### Astarte data types to BSON types
+
+| Astarte Data Type | BSON Type           | Size in Bytes                                      |
+|-------------------|---------------------|----------------------------------------------------|
+| double            | double (0x01)       | 8                                                  |
+| integer           | int32 (0x10)        | 4                                                  |
+| boolean           | boolean (0x08)      | 1                                                  |
+| longinteger       | int64 (0x12)        | 8                                                  |
+| string            | UTF-8 string (0x02) | >= length (encoding dependent)                     |
+| binaryblob        | binary (0x05)       | length                                             |
+| datetime          | UTC datetime (0x09) | 8                                                  |
+| doublearray       | Array (0x04)        | (8 + keysize) * count                              |
+| integerarray      | Array (0x04)        | (4 + keysize) * count                              |
+| booleanarray      | Array (0x04)        | (1 + keysize) * count                              |
+| longintegerarray  | Array (0x04)        | (1 + keysize) * count                              |
+| stringarray       | Array (0x04)        | depends on count, length, keys length and encoding |
+| binaryblobarray   | Array (0x4)         | depends on count, keys length and length           |
+
+Object aggregations are encoded as embedded documents.
+
+## Introspection
+
+Each devices must declare the set of supported interfaces and their version.
+This message is called "introspection" in Astarte jargon and it performed by publishing on the device root topic the list of supported interfaces.
+
+Introspection payload has the following format (in BNF like syntax):
+
+```
+introspection ::= introspection_list
+introspection_list ::= introspection_entry ";" introspection_list | introspection_entry
+introspection_entry ::= interface_name ":" interface_major_version ":" interface_minor_version
+```
+
+The following is a valid introspection payload:
+
+```
+com.example.MyInterface:1:0;org.example.DraftInterface:0:3
+```
+
+## Empty Cache
+
+A device may declare to have no knowledge about server owned set properties, this usually happens on a new device, after a factory reset or when cache is lost.
+In Astarte jargon this message is called "empty cache" and it is performed by publising "1" on the device ``/control/emptyCache`` topic.
```

Comment:
empty cache either italic or monospace. avoid using double quotes to express these concepts